.github/workflows/ccip-find-flaky-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: "Flaky test detection"
+
+on:
+  # Allow manual triggering via the GitHub UI
+  workflow_dispatch:
+  # Run the workflow every 4 hours
+  schedule:
+    - cron: "0 */4 * * *"
+
+jobs:
+  flaky-test-detection:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go-version: ['1.22']
+    defaults:
+      run:
+        working-directory: .
+    steps:
+      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      - name: Setup Go ${{ matrix.go-version }}
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Display Go version
+        run: go version
+      - name: Run tests 50 times.
+        run: TEST_COUNT=50 make test
+      - name: Notify the team by commenting on latest commit
+        if: failure()
+        run: |
+          COMMIT_SHA=$(git rev-parse HEAD)
+          curl -X POST \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/commits/$COMMIT_SHA/comments \
+            -d '{"body":"@smartcontractkit/ccip-offchain flaky tests were detected, please check."}'
```

Comment:
What commit is this? I think we need slack integration.

Reply:
👍 for slack notification, there is too much spam from Github already

Reply:
I don't have the ability anymore, we need to create a security ticket